src/toil/batchSystems/lsf.py

```diff
@@ -136,7 +136,7 @@ def getJobExitCode(self, lsfJobID):
                                  "{}".format(job))
                     return 0
                 elif line.find("Completed <exit>") > -1:
-                    logger.debug("Detected job failed for job: "
+                    logger.error("Detected job failed for job: "
```

Comment:
Since this backed extends `AbstractGridEngineBatchSystem` then all implementers of `getJobExitCode` should throw an exception instead of using `1` as the return in case the job itself failed to differentiate between job failure and the underlying tool having an exit code of `1`. I'm the most recent person to work on this code, so I can fix this tomorrow if you don't want to do it today.

Once the exception is thrown then we can have a unified `logger.error` in `AbstractGridEngineBatchSystem`.

Reply:
As long as we expose the `LSF` job id to the user through logging we are good. It doesn't really matter *when* it happens as long as you are reported with the failed LSF job id even if your logging level is not `debug` (so you can explore with `bsub -L` for instance).

Reply:
But logging to `error` the failed LSF job ID shouldn't affect the return value of the function... I'm not following here 

>Since this backed extends AbstractGridEngineBatchSystem then all implementers of getJobExitCode should throw an exception instead of using 1 as the return in case the job itself failed to differentiate between job failure and the underlying tool having an exit code of 1.

Reply:
Since my proposed change will touch all of the `AbstractGridEngineBatchSystem`s I'm going to do it in a separate PR. In the meantime I've accepted your suggestion to increase the logging level.